torch_xla/core/xla_model.py

```diff
@@ -137,7 +137,7 @@ def get_xla_supported_devices(devkind=None, max_devices=None):
     The list of device strings.
   """
   xla_devices = _DEVICES.value
-  devkind = [devkind] if devkind else ['TPU', 'GPU', 'CPU']
+  devkind = [devkind] if devkind else ['TPU', 'GPU', 'CPU', 'SPMD']
```

Comment:
For now, add a TODO comment to clarify that the flag interface doesn't need/use this -- if that's the case.

Reply:
I just removed this for now - since we may not even go this route.